startleft/iriusrisk.py

```diff
@@ -230,3 +230,76 @@ def validate_otm(self):
         logger.debug(f"--- Schema to validate against ---\n{self.schema.json()}\n--- End of schema ---")
         self.schema.validate(self.otm)
         return self.schema
+
+    def check_otm_ids(self):
+        all_valid_ids = []
+        repeated_ids = []
+        wrong_trustzone_ids = []
+        wrong_component_ids = []
+        wrong_component_parent_ids = []
+        wrong_dataflow_ids = []
+        wrong_dataflow_from_ids = []
+        wrong_dataflow_to_ids = []
+
+        for trustzone in self.otm['trustzones']:
+            if trustzone['id'].strip() == '':
+                wrong_trustzone_ids.append(trustzone['id'])
```

Comment:
Here we are adding empty values to the wrong_trustzone_ids, which won't be a good error message to log. Also I think that validations of required parameters as ids is currently done in the schema validation, so probably we shouldn't have to do it here at all

(Same comment components and data flows)

Reply:
I've just checked the schema and it doesn't fails on empty fields, it only fails when the node doesn't exists. Probably is better to adapt the schema validation to also fail on these situations.

Reply:
Changes done.